src/main/java/com/openlattice/postgres/PostgresTable.java

```diff
@@ -165,6 +165,34 @@
                             CATEGORY,
                             SHARDS );
     //.setUnique( NAMESPACE, NAME );
+    public static final PostgresTableDefinition        ENTITY_TYPE_COLLECTIONS      =
+            new PostgresTableDefinition( "entity_type_collections" )
+                    .addColumns(
+                            ID,
+                            NAMESPACE,
+                            NAME,
+                            TITLE,
+                            DESCRIPTION,
+                            SCHEMAS,
+                            TEMPLATE );
+    public static final PostgresTableDefinition        ENTITY_SET_COLLECTIONS       =
+            new PostgresTableDefinition( "entity_set_collections" )
+                    .addColumns(
+                            ID,
+                            NAME,
+                            TITLE,
+                            DESCRIPTION,
+                            CONTACTS,
+                            ENTITY_TYPE_COLLECTION_ID,
+                            ORGANIZATION_ID );
```

Comment:
Primary keys are missing for ENTITY_TYPE_COLLECTIONS and ENTITY_SET_COLLECTIONS.

Reply:
ID is primary key

Reply:
Ohh, you're right!